app/src/main/res/layout/content_main.xml

```diff
@@ -76,62 +77,9 @@
-
-            <LinearLayout
-                android:id="@+id/settings_linear_layout"
-                android:layout_width="@dimen/spacing_none"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="@dimen/layout_margin_large"
-                android:layout_marginLeft="@dimen/layout_margin_large"
-                android:layout_marginEnd="@dimen/layout_margin_large"
-                android:layout_marginRight="@dimen/layout_margin_large"
-                android:orientation="vertical"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintHorizontal_bias="1.0"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/textView3">
-
-                <CheckBox
-                    android:id="@+id/cb_simulation"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/simulation"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-                <CheckBox
-                    android:id="@+id/cb_load_resources_from_phone"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/layout_margin_medium"
-                    android:text="@string/load_from_phone"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-                <CheckBox
-                    android:id="@+id/cb_audio_feedback"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/layout_margin_medium"
-                    android:text="@string/audio_feedback"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-                <CheckBox
-                    android:id="@+id/cb_24bit"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/layout_margin_medium"
-                    android:text="@string/mode_24bit"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-                <CheckBox
-                    android:id="@+id/cb_advanced_mode"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/layout_margin_medium"
-                    android:text="@string/mode_advanced"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-            </LinearLayout>
+            android:layout_height="match_parent"
+            android:layout_marginTop="8dp"
```

Comment:
use dimens file for indirect referencing. avoid hardcoding.

Reply:
> use dimens file for indirect referencing. avoid hardcoding.

Yeah just was waiting for a review on the design. Will do these asap.